cmd/kusk/README.md

```diff
@@ -0,0 +1,300 @@
+|   `--upstream.port`    |                       port that upstream service is exposed on (default: 80)                        |     ❌     |
+|  `--envoyfleet.name`   |                               name of envoyfleet to use for this API                                |     ✅     |
+| `envoyfleet.namespace` |                  namespace of envoyfleet to use for this API. Default: kusk-system                  |     ❌     |
+
+### Example
+
+Take a look at the [`./examples/httpbin-spec.yaml`](./examples/httpbin-spec.yaml)
+
+```sh
+kusk api generate -i ./examples/httpbin-spec.yaml --name httpbin-api --upstream.service httpbin --upstream.port 8080 --envoyfleet.name kusk-gateway-envoy-fleet
+```
+
+The output should contain the following x-kusk extension at the top level
+
+```sh
+...
+x-kusk:
+  cors: {}
+  path:
+    rewrite:
+      pattern: ""
+      substitution: ""
+  upstream:
+    service:
+	name: httpbin
+	namespace: default
+	port: 8080
+```
+
+## dashboard
+Access the kusk dashboard. kusk dashboard will start a port-forward session on port 8080 to the envoyfleet
+serving the dashboard and will open the dashboard in the browser. By default this is kusk-gateway-private-envoy-fleet.kusk-system.
+
+The flags --envoyfleet.namespace and --envoyfleet.name can be used to change the envoyfleet.
+
+### Flags
+|           Flag           |                                         Description                                          | Required? |
+| :----------------------: | :------------------------------------------------------------------------------------------: | :-------: |
+|      `--kubeconfig`      |                                 absolute path to kube config                                 |     ❌     |
+|   `--envoyfleet.name`    | kusk gateway dashboard envoy fleet service name. (default: kusk-gateway-private-envoy-fleet) |     ❌     |
+| `--envoyfleet.namespace` |         kusk gateway dashboard envoy fleet service namespace. (default: kusk-system)         |     ❌     |
+|    `--external-port`     |                    external port to access dashboard at. (default: 8080)                     |     ❌     |
+
+### Examples
+
+```sh
+$ kusk dashboard
+```
+
+Opens the kusk gateway dashboard in the browser by exposing the default private envoy fleet on port 8080
+
+```sh
+$ kusk dashboard --envoyfleet.namespace=other-namespace --envoyfleet.name=other-envoy-fleet
+```
+
+Specify other envoyfleet and namespace that is serving the dashboard
+
+```sh
+$ kusk dashboard --external-port=9090
+```
+
+Expose dashboard on port 9090
+
+```sh
+$ kusk dashboard --kubeconfig=/path/to/kube/config
+```
+
+Specify path to kube config. $HOME/.kube/config is used by default.
+
+# Installation
+
+## Homebrew
+
+```sh
+brew install kubeshop/kusk/kusk
+```
+
+## Go install the latest release on Github
+
+```sh
+go install github.com/kubeshop/kusk@latest
```

Comment:
No longer valid? Better yet, please remove all references to `github.com/kubeshop/kusk` since the repository is no longer needed right?